hadoop-shaded-protobuf/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+     Copyright [2019 - 2019] Confluent Inc.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>kafka-connect-storage-common-parent</artifactId>
+        <version>10.0.15-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kafka-connect-storage-common-hadoop-shaded-protobuf</artifactId>
+    <name>Kafka Connect Storage Common Source's version of Apache Hadoop's third-party shaded Protobuf</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <shaded.prefix>org.apache.hadoop.thirdparty</shaded.prefix>
+        <protobuf.shade.prefix>${shaded.prefix}.protobuf</protobuf.shade.prefix>
+        <protobuf.version>3.21.1</protobuf.version>
```

Comment:
is it possible to re-use protobuf.version from main pom of the repo - https://github.com/confluentinc/kafka-connect-storage-common/blob/ee74e7074ac995fcd7abc462bfe56da091aa1862/pom.xml#L93